src/Component/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Comments } from '../../Types';
+import './Style.css';
+
+type Props = {
+  comment: Comments,
+};
```

Comment:
```suggestion
type Props = {
  comment: Comment,
};
```
якщо поле називається comment, то чому інтерфейс - Comments?